src/Eurofurence.App.Domain.Model/ArtistsAlley/UserStatus/ArtistAlleyUserPenaltyRecord.cs

```diff
@@ -0,0 +1,45 @@
+using System.Collections.Generic;
+using System.ComponentModel.DataAnnotations.Schema;
+
+namespace Eurofurence.App.Domain.Model.ArtistsAlley
+{
+
+    
+    /// <summary>
+    /// Record for storing
+    /// </summary>
+    [Table("ArtistAlleyUserPenalties")]
+    public class ArtistAlleyUserPenaltyRecord : EntityBase
+    {
+
+        public ArtistAlleyUserPenaltyRecord()
+        {
+            Penalty = UserPenalties.OK;
+        }
+
+        /// <summary>
+        /// Possible penalties of a user
+        /// </summary>
+        public enum UserPenalties
+        {
+            BANNED = 0,
+            OK = 1
+        }
+
+        /// <summary>
+        /// Id of the user from the reg system
+        /// </summary>
+        [Column("UserId")]
+        public string UserId { get; set; }
+        
+        /// <summary>
+        /// The current status of the user
+        /// </summary>
+        [Column("Penalty")]
+        public UserPenalties Penalty { get; set; }
```

Comment:
Penalty -> PenaltyStatus or Status
I'd suggest just "Status" since we're already looking at the ArtistAlleyUserPenaltyRecord